spec/parsers/cr2_parser_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'spec_helper'
+
+describe FormatParser::CR2Parser do
+  describe 'is able to parse CR2 files' do
+    Dir.glob(fixtures_dir + '/CR2/*.CR2').each do |cr2_path|
+      it "is able to parse #{File.basename(cr2_path)}" do
+        parsed = subject.call(File.open(cr2_path, 'rb'))
+
+        expect(parsed).not_to be_nil
+        expect(parsed.nature).to eq(:image)
+        expect(parsed.format).to eq(:cr2)
+
+        expect(parsed.width_px).to be_kind_of(Integer)
+        expect(parsed.width_px).to be > 0
+
+        expect(parsed.height_px).to be_kind_of(Integer)
+        expect(parsed.height_px).to be > 0
+
+        expect(parsed.resolution).to be_kind_of(Integer)
+        expect(parsed.resolution).to be > 0
```

Comment:
In the tags we _do_ read, is there any useful information that is small and can be passed as, say, a Hash, into `intrinsics`? (also see on `Image#resolution` above)

Reply:
Camera model, shoot date, exposure and aperture values are provided in intrinsics, unfortunately I wasn't able to provide ISO number because some models are missing the ISO number tag.